presto-docs/src/main/sphinx/cache/local.rst

```diff
@@ -0,0 +1,69 @@
+=====================
+Alluxio Cache Library
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+Presto supports caching input data locally with a built-in cache
+to reduce query latency using Hive Connector.
+This cache, based on Alluixo client-side cache library,
+utilizes the local storage (such as SSD) with configurable capacity and locations.
+To understand its internals and the benchmark results of latency improvement,
+please read this `article <https://prestodb.io/blog/2020/06/16/alluxio-datacaching>`_.
+Note that this is a **read-cache**, **completely transparent** from users
+and **fully managed** by individual Presto workers.
+To provide Presto an independent, distributed cache service for read/write workloads with
+customized data caching policies, please refer to :doc:`/cache/service`.
+
+
+Setup
+-----
+
+To turn on this Alluxio local cache is very easy.
+Simply include the following configuration in ``etc/catalog/hive.properties``
+and restart Presto coordinator and workers:
+
+.. code-block:: none
+
+    hive.node-selection-strategy=SOFT_AFFINITY
+    cache.enabled=true
+    cache.type=ALLUXIO
+    cache.alluxio.max-cache-size=500GB
+    cache.base-directory=/tmp/alluxio-cache
+
+In the above example configuration,
+
+* ``hive.node-selection-strategy=SOFT_AFFINITY`` instructs Presto scheduler to take data affinity
```

Comment:
It's not clear which config belongs to the coordinator, and which ones to the worker. We should create separate blocks for coordinator and workers.

Reply:
You can refer to the blog if you need more info.

Reply:
already included in the doc